src/System.Private.Reflection.Core/src/System/Reflection/Runtime/General/MetadataReaderExtensions.NativeFormat.cs

```diff
@@ -326,49 +326,61 @@ public static Object ParseConstantValue(this Handle handle, MetadataReader reade
             return value;
         }
 
+        public static Object ParseConstantNumericValue(this Handle handle, MetadataReader reader)
```

Comment:
Factoring this into a separate method without having use for it does not look like an improvement to me.